damnit/api.py

```diff
@@ -171,6 +173,110 @@ def summary(self):
                 return blob2complex(value)
             return value
 
+    def preview_data(self, *, data_fallback=True, deserialize_plotly=True):
+        """Get the preview data for the variable
+
+        May return a 1D or 2D data array, a 3D RGB(A) arrray, a plotly figure
+        object, a str of plotly JSON (with deserialize_plotly=False) or None
```

Comment:
Very nitpicky, sorry:
```suggestion
        May return a 1D or 2D data array, a 3D RGB(A) arrray, a Plotly figure
        object, a str of Plotly JSON (with deserialize_plotly=False) or None
```